### DISCUSSION TITLE
How is a repository deleted?

### DISCUSSION THREAD

**Discussion by associateone** [community/community#75572] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have fought this off and on to frustration ever since creating the first repository on my account. Short of using command line git, which I am uneducated about, I can find no hint anywhere for how to remove/delete a repository. Can anyone help?

**Comment by DNin01** [comment]
Here: https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository

**Comment by DNin01** [comment]
If you want to preserve the repository but not work on it anymore, you can also [archive it](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories).

**Comment by kirubel606** [comment]
1. **Navigate to the Repository:**
   Go to the main page of the repository you want to delete on GitHub.

2. **Settings:**
   Click on the "Settings" tab, located towards the right side of the repository menu.

3. **Scroll Down to Danger Zone:**
   Scroll down to the "Danger Zone" section at the bottom of the Settings page.

4. **Delete This Repository:**
   Find the "Delete this repository" option in the Danger Zone.

5. **Confirm Repository Name:**
   You will be prompted to enter the name of the repository to confirm the deletion. Enter the repository's name.

6. **Delete Repository:**
   Click the "I understand the consequences, delete this repository" button.

7. **Authentication:**
   If required, authenticate with your GitHub password or through any other verification method.

Please be aware that deleting a repository is irreversible, and all data associated with the repository, including issues, pull requests, and wiki pages, will be permanently removed. Make sure you have a backup of any important information before proceeding.

**Comment by associateone** [comment]
Thank you for your response, it helps immensely.

On Tue, Nov 14, 2023 at 6:34 PM DNin01 ***@***.***> wrote:

> If you want to preserve the repository but not work on it anymore, you can
> also archive it
> <https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories>
> .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/75572#discussioncomment-7571841>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6WIIELCTM5APBGKKVRLXLYEQS2RAVCNFSM6AAAAAA7LRT6CCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNZRHA2DC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Steve Brooks
14777 Palm Dr. #51
Desert Hot Springs, Ca 92240
316-207-8512
